docs/language.rst

```diff
@@ -195,10 +195,28 @@ value. It can have the following forms
         | value
         | value ('>' | '>=' | '<' | '<=' | '==' | '!=') value
         | value 'in' value
+        | value 'not in' value
         | functioncall
         | value 'is' 'defined'
         ;
 
+
+The ``in`` and ``not in`` operators can be used to check if a value is present in a list:
+
+.. code-block:: inmanta
+
+    myfiles = ["/a/b/c", "/c/d/e", "x/y/z/u/v/w"]
+
+    condition1 = "/a/b/c" in myfiles # evaluates to True
+    condition2 = "/f/g/h" in myfiles # evaluates to False
+
+    condition3 = "/a/b/c" not in myfiles # evaluates to False
+    condition4 = "/f/g/h" not in myfiles # evaluates to True
+
+    condition5 = not "/a/b/c" in myfiles # evaluates to False
+    condition6 = not "/f/g/h" in myfiles # evaluates to True
```

Comment:
```suggestion
    condition1 = "/a/b/c" in myfiles # evaluates to true
    condition2 = "/f/g/h" in myfiles # evaluates to false

    condition3 = "/a/b/c" not in myfiles # evaluates to false
    condition4 = "/f/g/h" not in myfiles # evaluates to true

    condition5 = not ("/a/b/c" in myfiles) # evaluates to false
    condition6 = not ("/f/g/h" in myfiles) # evaluates to true
```
The parentheses aren't strictly required but I think it helps readability here to be explicit.